src/components/Bullet/index.tsx

```diff
@@ -0,0 +1,128 @@
+import { Theme, css } from '@emotion/react'
+import styled from '@emotion/styled'
+import PropTypes from 'prop-types'
+import React, { ComponentProps } from 'react'
+import Icon from '../Icon'
+import Tooltip from '../Tooltip'
+
+const variants = {
+  danger: ({ theme }: { theme: Theme }) => `
+    background-color: ${theme.colors.danger.background};
+    color: ${theme.colors.danger.text};
+  `,
+  default: ({ theme }: { theme: Theme }) => `
+    border: 1px solid ${theme.colors.neutral.borderWeak};
+    background-color: ${theme.colors.neutral.background};
+    color: ${theme.colors.neutral.text};
+  `,
+  disabled: ({ theme }: { theme: Theme }) => `
+    border: 1px solid ${theme.colors.neutral.borderWeak};
+    background-color: ${theme.colors.neutral.background};
+    color: ${theme.colors.neutral.textWeak};
+  `,
+  info: ({ theme }: { theme: Theme }) => `
+    background-color: ${theme.colors.info.background};
+    color: ${theme.colors.info.text};
+  `,
+  primary: ({ theme }: { theme: Theme }) => `
+    background-color: ${theme.colors.primary.background};
+    color: ${theme.colors.primary.text};
+  `,
+  success: ({ theme }: { theme: Theme }) => `
+    background-color: ${theme.colors.success.background};
+    color: ${theme.colors.success.text};
+  `,
+  warning: ({ theme }: { theme: Theme }) => `
+    background-color: ${theme.colors.warning.background};
+    color: ${theme.colors.warning.text};
+  `,
+} as const
+type BulletVariant = keyof typeof variants
+export const bulletVariants = Object.keys(variants) as BulletVariant[]
+
+const sizes = {
+  medium: css`
+    width: 32px;
+    height: 32px;
+    font-size: 16px;
+  `,
+  small: css`
+    width: 24px;
+    height: 24px;
+    font-size: 14px;
+  `,
+} as const
```

Comment:
```suggestion
const sizes = {
  medium: `
    width: 32px;
    height: 32px;
    font-size: 16px;
  `,
  small: `
    width: 24px;
    height: 24px;
    font-size: 14px;
  `,
} as const
```

doesn't it work this way ?